Library/Formula/truncate.rb

```diff
@@ -0,0 +1,18 @@
+class Truncate < Formula
+  desc "truncates a file to a given size"
+  homepage "http://www.vanheusden.com/truncate"
+  url "https://github.com/flok99/truncate/archive/fc30a29dab789249f0c5f6863fdc5c8d4c9c3f69.zip"
```

Comment:
I need to read more apparently; light tags are fine, but you have to push them as tags rather than just `git push`. That may have been the error here. Apologies for any confusion.

The new tag looks great. The Makefile says version `0.9` and the tag is `0.8` but we can self-correct that in the formula if necessary.